packages/pilot/src/components/MetricChart/charts/RadialChart/Radial.js

```diff
@@ -0,0 +1,100 @@
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+import {
+  evolve,
+  juxt,
+  uncurryN,
+} from 'ramda'
+import {
+  LabelList,
+  PolarAngleAxis,
+  RadialBar,
+  RadialBarChart,
+  ResponsiveContainer,
+} from 'recharts'
+
+import CustomLabel from './CustomLabel'
+
+import sizePropValidation from '../../sizePropValidation'
+
+const buildChartData = uncurryN(2, colors => juxt([
+  evolve({
+    fill: dataColor => dataColor || colors.fill,
+  }),
+]))
+
+const Radial = ({
+  data,
+  labelFormatter,
+  styles: {
+    colors = {
+      fill: '#2bb1d1',
+    },
+    height = 120,
+    innerRadius = 55,
+    outerRadius = '80%',
```

Comment:
acho que 100% ficaria mais apresentável heheh
```suggestion
    outerRadius = '100%',
```